rascal-vscode-extension/src/test/vscode-suite/dsl.test.ts

```diff
@@ -103,6 +101,10 @@ parameterizedDescribe(function (errorRecovery: boolean) {
         await fs.writeFile(TestWorkspace.picoFile, picoFileBackup);
     });
 
+    afterEach("print output in case of test failure",
+        async function () { await printRascalOutputOnFailure(this, ide, "Language Parametric Rascal"); }
```

Comment:
`async` and a single `await`  can also just be `function(){ return ...; }` which is the same, with less keywords.